src/app/public/modules/autocomplete/autocomplete.component.ts

```diff
@@ -148,6 +148,7 @@ export class SkyAutocompleteComponent
   private isMouseEnter = false;
   private searchResultsIndex = 0;
   private searchText: string;
+  private previousValue: string;
```

Comment:
Storing the `ControlValueAccessor`'s previous value within the wrapping component feels off to me. The autocomplete component (in theory) should only be dealing with the search text and the dropdown. The value is handled by the `SkyAutocompleteDirective`. Any way we can isolate value checks to the directive?

Reply:
I also noticed that the input directive watches for blur events, if that's useful:
https://github.com/blackbaud/skyux-lookup/blob/master/src/app/public/modules/autocomplete/autocomplete-input.directive.ts#L89-L94

Reply:
Hmm, valid concern. I started here, but then realized the `SkyAutocompleteComponent` only emits the event when a search selection is made. No matter what the direction we go in, we'll need this component to be aware of blur and emit an event if the text is cleared. That's why I landed on this solution. Without storing the previous version, the emitter would fire on every blur, regardless if the value had changed.

Ideas on a different approach?

Reply:
We could add a new output event to the directive, like `reset` or `cleared`...

Reply:
I approached this a different way. What do you think?